app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -262,6 +267,8 @@ private void setupTimelinePreferences() {
             filterRemoveRegexMatcher = Pattern.compile(regexFilter, Pattern.CASE_INSENSITIVE)
                     .matcher("");
         }
+
+        collapseLongStatusContent = preferences.getBoolean("collapseLongStatuses", true);
```

Comment:
Extract preference key constant please

Reply:
Is there a class that is responsible for storing these accessible by TimelineFragment? The previews preferences are also setup like this, it's where I looked to decide how to do it.
Should I move them all to the `notranslate.xml` resource file and use `getString()` instead?

Reply:
I'll just open a new PR later and take care of moving all the string resources to the `notranslate.xml` if you prefer, to not pollute this PR with changes regarding other configurations, since if I do it for mine I might as well do it for all of them.

Reply:
Sounds great, thanks